package.json

```diff
@@ -7,14 +7,15 @@
     "dev": "echo \"Use the 'yarn go' command to develop locally.\"",
     "build": "NODE_ENV=test yarn run test && NODE_ENV=production yarn run relay && next build",
```

Comment:
I think we'll want to add yarn run lint to this build command

Reply:
Good call

Reply:
Yes, that sounds good.